tooling/miniutil/src/build/terminator.rs

```diff
@@ -96,7 +98,26 @@ impl FunctionBuilder {
         unwind_block: BbName,
     ) {
         let next_block = self.declare_block();
-        self.handle_call(ret, f, args, Some(next_block), Some(unwind_block));
+        self.handle_call(
+            ret,
+            f,
+            args,
+            CallingConvention::Rust,
+            Some(next_block),
+            Some(unwind_block),
+        );
+    }
+
+    /// Call a function using C calling conventions.
+    pub fn call_extern(
```

Comment:
I'd call this `call_with_conv` and then pass it the expected calling convention as argument.